Makefile

```diff
@@ -134,6 +135,15 @@ thinkhazard/static/build/%.css: $(LESS_FILES) .build/node_modules.timestamp
 	sed -e 's#{{PYTHONPATH}}#$(shell .build/venv/bin/python -c "import distutils; print(distutils.sysconfig.get_python_lib())")#' \
 		-e 's#{{WSGISCRIPT}}#$(abspath .build/venv/thinkhazard.wsgi)#' $< > $@
 
+.build/pyramid.zip:
+	mkdir -p $(dir $@)
+	wget -O $@ https://github.com/Pylons/pyramid/archive/master.zip
```

Comment:
> Is it safe to get pyramid from master instead of a tag?

You're right, it would be better to use a fixed version. I'll change it before merging. Thanks.